tamr_unify_client/models/project/mastering.py

```diff
@@ -76,4 +76,18 @@ def published_clusters(self):
         alias = self.api_path + "/publishedClusters"
         return Dataset(self.client, None, alias)
 
+    def record_clusters_with_data(self):
+        """Record clusters with data.
+
+        Call :func:`~tamr_unify_client.models.dataset.resource.Dataset.refresh` from
+        this dataset to generate clusters with data.
+
+        :returns: Record clusters with its corresponding data (i.e. relative_id, name, version).
```

Comment:
For this line, something like "The record clusters with data represented as a dataset" might be better, just because the other methods in mastering.py follow this format